### ISSUE TITLE
return 00:00:00:00:00:00 sometimes 

### ISSUE THREAD

**Issue by ihabhamad** [scravy/node-macaddress#22]
my friend if i used .all function sometimes return to me 00:00:00:00:00:00 on some users

**Comment by djedlajn** [comment]
This should be an issue with libuv c++ library. I think that patch has landed.

**Comment by osenvosem** [comment]
The module always returns zeroes on all interfaces on macOs.
```
{ en0:
     { ipv6: 'fe80::100f:5aa3:96e0:937',
       mac: '00:00:00:00:00:00',
       ipv4: '10.1.1.83' },
    awdl0:
     { ipv6: 'fe80::5c42:d3ff:fe9c:b379', mac: '00:00:00:00:00:00' },
    utun0:
     { ipv6: 'fe80::26d7:ed97:caf4:3d91', mac: '00:00:00:00:00:00' } }
```

**Comment by sheshnath08** [comment]
I was able to fix this issue by changing this if the condition on line https://github.com/scravy/node-macaddress/blob/master/index.js#L134 to
`
if (!ifaces[iface].mac || ifaces[iface].mac === '00:00:00:00:00:00') 
`

**Comment by scravy** [comment]
@sheshnath08 I have incorporated the check for a mac address comprised of all zeroes in 0f329b4, has been released along `v0.4.3`
